src/main/java/mat/client/export/measure/ManageMeasureExportView.java

```diff
@@ -132,11 +132,12 @@ public void setExportOptionsBasedOnVersion(String releaseVersion, boolean isComp
 			vp.add(jsonRadio);
 			vp.add(humanReadableRadio);
 			vp.add(allRadio);
-			// Transfer to Madie is displayed only for the measure owner
-			if (currentMeasureOwnerId != null && MatContext.get().getFeatureFlagStatus(FeatureFlagConstant.MADIE)
-					&& currentMeasureOwnerId.equals(MatContext.get().getCurrentUserInfo().userId)) {
+			if (MatContext.get().getFeatureFlagStatus(FeatureFlagConstant.MADIE)) {
 				vp.add(transferToMadieRadio);
 			}
+			if (!isTransferableToMadie) {
+				transferToMadieRadio.setEnabled(false);
```

Comment:
Any concern with altering the enabled state if the radio button isn't displayed?

I'm guessing not.

Reply:
I guess not